.circleci/conditional-builds.sh

```diff
@@ -0,0 +1,255 @@
+#! /usr/bin/env bash
+
+set -e
```

Comment:
I think it would be nice to add a comment here explaining where this code is from

Reply:
Yes! I'll do that.